src/components/ButtonPanel.js

```diff
@@ -1,39 +1,46 @@
 import React from 'react';
+import propTypes from 'prop-types';
 import Button from './Button';
 
-function ButtonPanel() {
+function ButtonPanel(props) {
+  const { clickHandler } = props;
   return (
```

Comment:
* [ ] ButtonPanel should have `handleClick` function implemented ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png))

```javascript
handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
};
```